.travis.yml

```diff
@@ -0,0 +1,48 @@
+---
+language: php
+
+php:
+  - hhvm
+  - 7.1
+  - 7.0
+  - 5.6
+  - 5.5
+  - 5.4
+  - 5.3
+
+# This triggers builds to run on the new (faster) TravisCI infrastructure.
+# See: http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+sudo: false
+
+cache:
+  directories:
+    - "${HOME}/.composer/cache"
+
+env:
+  global:
+    - PATH="${HOME}/bin:${PATH}"
+
+matrix:
+  allow_failures:
+  - php: hhvm
+  - php: 5.5
+  - php: 5.4
+  - php: 5.3
+  fast_finish: true
+
+before_install:
+  - npm set loglevel error
+  - npm set progress false
+
+install:
+  - composer global require "dealerdirect/qa-tools:*"
```

Comment:
![Patrick](https://dealerdirect.github.io/Memes/img/png/patrick.png)

Yeah... I must have not been thinking straight (or too lazy).

Will improve the merge-request to fix this.